### ISSUE TITLE
Paramount_Botulus - Botulus_Rex - Gallu wrong tp moves

### ISSUE THREAD

**Issue by gori123** [LegionDark/Issues#1546]
**Date & Time**: 15.22

**FFXI Client Version (use `/ver`)**: 30170601

**Server's Expected Client Version matches, yes/no? (use `$ecv`)**: yes

**Character Name**: Tonton

**Nation**: sandoria

**Job(level)/Sub Job(level)**: 99rdm/49whm

**NPC or Monster or item Name**: Botulus_Rex - Paramount_Botulus -Paramount Gallu - Ig-Alima

**Zone name**:  legion - valkrum dunes - buburimu

**Coordinates (use `$where`)**:  multiple

**ffxiah.com link (for items issues only)**: 
https://www.bg-wiki.com/bg/Ig-Alima
https://www.bg-wiki.com/bg/Paramount_Gallu
https://www.bg-wiki.com/index.php/Paramount_Botulus
https://www.bg-wiki.com/bg/Botulus_Rex
**Multi-boxing? (multiple clients on same connection)**: yes

**Steps To Reproduce / any other info**:  These mobs does not have str down , vit-down , dex-down, agi-down, int-down, mnd-down and chr-down effect. (not sure Botulus casts magic def down and def down) These moves causes very cheap deaths with other dangerous moves these mobs have

Sorry if i am wrong since i have not played in retail.

**Comment by R3P0FFXI** [comment]
@gori123 I can verify these NM do have -stats effect according to retail

    Oblivion's Mantle: AoE Weakness (1 min. duration) and 10-count Doom.
        Players killed by the doom effect will return with double weakness due to the initial weakness effect.
        Will always use Oblivion's Mantle right after Diluvial Wake.  
Bolt of Perdition: AoE physical Lightning damage, knockback, Mute and Amnesia (30~40 sec).
Searing Halitus: AoE physical Fire damage.
Crippling Rime: Conal physical Ice damage.
    Diluvial Wake: Conal physical Water damage, all stats down, Defense Down, and Magic Defense Down.
    Divesting Gale: AoE physical Wind damage, Encumberance and Muddle.
    Kurnugi Collapse: AoE physical Earth damage, Magic Accuracy Down, Accuracy Down, and Burn (38 hp/tick). 

@TeoTwawki it appears some status effects are wrong in certain scripts, Crippling Rime Currently has all - stats when it should be Diluvual Wake, looks like we may need to adjust at some point to get them a bit more accurate.

**Comment by Heepster03** [comment]
Pick and Choose. Clopedia and BGwiki don't agree. 

It appears as if ours might've been coded after BGwiki, which is what I posted below:

Crippling Rime: Conal ice damage, all attributes down.
Diluvial Wake: Area of Effect water damage, Defense Down, and Magic Defense Down

**Comment by TeoTwawki** [comment]
BG wiki is usually more accurate and ffxiclopedia will often have wild guesses tossed in. When they disagree I go with BG unless I have strong evidence to the contrary.

**Comment by gori123** [comment]
my opinion is not important but i saw my mistake,
-gallu does not need fixing but
-botulus rex and paramount botulus do need

**Comment by TeoTwawki** [comment]
> my opinion is not important

I really hate it when people say that. Like want to shake them by the shoulders and tell them to stop the self-putdown stuff and to go look in a mirror and recite Stuart Smalley's mantra :frowning_face: https://youtu.be/-DIETlxquzY

tl;dr
all player opinions matter, though they often call things opinions which aren't

**Comment by R3P0FFXI** [comment]
@TeoTwawki while were going through mobskills please advise
